fitwsarah_frontend/src/views/AdminPanelPage/AdminAppointments.js

```diff
@@ -142,49 +142,83 @@ function AdminAccounts() {
         }
     };
 
+    const handleAcceptedAppointment = (appointmentId) => {
+        const isConfirmed = window.confirm("Are you sure you want to accept this appointment?");
+        if (isConfirmed) {
+            handleAppointmentRequest(appointmentId, 'scheduled');
+        }
+    };
+
 
     const getAppointments = () => {
         const params = new URLSearchParams();
         searchTerm.forEach(term => {
-          if (term[1]) {
-              params.append(term[0], term[1]);
-          }
+            if (term[1]) {
+                params.append(term[0], term[1]);
+            }
         });
 
-        fetch(`${process.env.REACT_APP_BASE_URL}/api/v1/appointments${params.toString() && "?" + params.toString()}`, {
+        fetch(`http://localhost:8080/api/v1/appointments${params.toString() && "?" + params.toString()}`, {
```

Comment:
You forgot to chanhe the url to ${process.env.REACT_APP_BASE_URL}

Reply:
Oups thanks for reminding me i will change it.